mssqlcli/jsonrpc/contracts/tests/test_json_rpc_contracts.py

```diff
@@ -214,7 +219,8 @@ def verify_query_service_response(self,
                         query_batch_summaries = len(response.batch_summaries)
                         query_result_set_summaries = len(
                             response.batch_summaries[0].result_set_summaries)
-                        query_row_count = response.batch_summaries[0].result_set_summaries[0].row_count
+                        query_row_count = response.batch_summaries[0].result_set_summaries[0]\

```

Comment:
Why this one? Is there a max line length rule?

Reply:
Yep